molecule/external-secrets-operator/verify.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+- name: Verify
+  hosts: "{{ target | default('all') }}"
+  tasks:
+    - name: Apply Fake ClusterSecretStore and ExternalSecret
+      run_once: true  # noqa: run-once[task]
+      kubernetes.core.k8s:
+        state: present
+        definition: "{{ lookup('file', '../../' + item) | from_yaml }}"
+        apply: true
+        server_side_apply:
+          field_manager: atmosphere
+          force_conflicts: true
+      loop:
+        - tests/manifests/fake-clustersecretstore.yaml
+        - tests/manifests/fake-externalsecret.yaml
```

Comment:
I think we can move these into the scenario folder, so it's `molecule/external-secrets-operator/manifests` or something like that so it's enclosed. :)